overlord/servicestate/servicestate.go

```diff
@@ -278,6 +283,19 @@ func (sd *StatusDecorator) DecorateWithStatus(appInfo *client.AppInfo, snapApp *
 			})
 		}
 	}
+	// Decorate with D-Bus names that activate this service
+	for _, slot := range snapApp.ActivatesOn {
+		var busName string
+		if err := slot.Attr("name", &busName); err != nil {
+			return fmt.Errorf("cannot get D-Bus bus name of slot %q: %v", slot.Name, err)
+		}
+		appInfo.Activators = append(appInfo.Activators, client.AppActivator{
+			Name:    busName,
+			Enabled: true,
+			Active:  true,
```

Comment:
hmm so is this because there is no way to disable the D-Bus activation of a service? It seems to be so, since we use `BusName=` to implement D-Bus activation AFAICT, and the only way to disable D-Bus activation would be to remove `BusName=` from the systemd service and reload it. Is that an accurate understanding of the situation?

Reply:
`BusName=` in the systemd unit file is not related to activation.  Instead it is used in combination with `Type=dbus` to determine ready state: after launching the service, it will be considered to be in the "starting" state until someone registers the specified name on the system bus (or session bus if this is a user scope unit).  This is an alternative to using e.g. `Type=notify`.

If we wanted to support enabling/disabling the activator, that would require creating or deleting the service activation files under `/var/lib/snapd/dbus-1`.  Currently we always create these files, so I'm always marking the activator as enabled/active.

Reply:
I see, thanks for explaining, could you add a comment to that effect here explaining why they are always active/enabled then?